helix-term/src/ui/statusline.rs

```diff
@@ -354,6 +355,19 @@ where
     write(context, title, None);
 }
 
+fn render_read_only_indicator<F>(context: &mut RenderContext, write: F)
+where
+    F: Fn(&mut RenderContext, String, Option<Style>) + Copy,
+{
+    let readonly = if context.doc.write_permission == WritePermission::ReadOnly {
+        context.editor.config().read_only_indicator.to_string()
+    } else {
+        String::new()
+    };
+
+    write(context, readonly, None);
```

Comment:
It's a nit but I think the `readonly` variable should be `indicator` instead. The `readonly` name makes me automatically think it's a bool 🙃

Reply:
how about `readonly_indicator`? There may be other indicators later.